specifications/api/swagger.yaml

```
@@ -2097,6 +2105,7 @@ components:
         mint: *transactionMint
         status: *transactionStatus
         metadata: *transactionMetadata
+        is_valid_script: *transactionScriptValidity
```

Suggestion: use `script_validity` here:
```suggestion
        script_validity: *transactionScriptValidity
```

---

062528a